EnglishFilter/sections/general_extensions.txt

```diff
@@ -229,6 +229,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! pop-ups in uploader.*, zeefiles.*
```

Comment:
Please provide some examples for using `*`.

Reply:
I didn't look up but I think they have more domains.

Reply:
> I didn't look up but I think they have more domains.

That's not enough. We need real examples.
For this PR just use original domains without `*`.

Reply:
Ok.

Reply:
@zloyden Swapped all.